src/platform/testing/e2e/nightwatch-commands/allyCheckTabbableCount.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+
+const ally = require('ally.js');
+
+/**
+ * Checks for the number of tabbable elements on the page,
+ * using the ally.query.tabbable method:
+ * https://allyjs.io/api/query/focusable.html
+ *
+ * "Tabbable" means the element or elements can receive
+ * keyboard focus natively by pressing `TAB` or `SHIFT + TAB`.
+ *
+ * @method allyCheckTabbableCount
+ * @param {string} [selector] The selector (CSS / Xpath) used to locate the element.
+ * @param {function} [callback] Optional callback function to be called when the command finishes.
+ * @returns {array} The array of tabbable elements found in the selector.
+ * @api commands
+ */
+exports.command = function allyCheckTabbableCount(selector, callback) {
+  return this.execute(
+    function(sel) {
```

Comment:
Refactor this to an `() =>` arrow function

Reply:
You may or may not run into trouble with this. Specifically, [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) change the normal rebinding of `this`. Maybe it'll be fine, but if you go to refactor it and run into unexpected issues, that might be why.

Also just throwing it out there: Just because it _can_ be changed to be "sleeker" doesn't mean it necessarily _should_ be. Having `function` declarations is totally fine, so don't feel like you need to in order for this to be "clean."

Reply:
That's true. I think it was more about being consistent. I had one arrow function in the group and it worked okay, but yeah, moving back to `function(sel)` doesn't rebind `this`, and that may be more beneficial.